src/compiler/types.ts

```diff
@@ -2037,6 +2037,10 @@ namespace ts {
         end: -1;
     }
 
+    export interface RegionRange extends TextRange {

```

Comment:
should this be used in the definition of `OutliningSpan` now?

Reply:
also please move this to `services/types.ts` instead.

Reply:
i see now this is only used internally while building the spans. then this should move to `src/services/outliningElementsCollector.ts` and not be exported.

Reply:
Moved.